tests/unit/test_utils.py

```diff
@@ -329,3 +329,16 @@ def test_check_before_after_census_day(self):
         self.after_census_day(2021, 3, 22)
         self.before_census_day(2021, 3, 21)
         self.before_census_day(2021, 3, 20)
+
+    def test_client_ip(self):
```

Comment:
Consider enhancing for:
- inputs with and without various whitespace
- inputs with no client_ip in the request
- inputs with just one client_ip in the request

Reply:
I've added a couple more tests, and got rid of the whitespace code (I realised I was doing the split incorrectly)